lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/EagerFallback.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/ivalue.h>
+#include <ATen/core/stack.h>
+
+namespace torch_lazy_tensors {
+
+void eager_fallback(const c10::OperatorHandle& op, torch::jit::Stack* stack,
```

Comment:
i briefly thought about making this a more general function, but it is obviously coupled to backend details for good reason; is there anything about it we should try to promote to an interface in the future, or provide shared infra for?

Reply:
I've asked myself the same question and don't know the answer. We can guess though, what else is going to have eager backends for the foreseeable future and be available on machines to which accelerators are connected? I think it's a safe bet to say CPU and GPU (CUDA / HIP would both be covered by this). Moreover, I suspect machines with non-GPU accelerators won't have GPUs, so the GPU path itself is only for GPU all-around.